src/Administration/Resources/app/administration/src/module/sw-order/view/sw-order-create-general/sw-order-create-general.html.twig

```diff
@@ -16,23 +16,21 @@
         position-identifier="sw-order-create-base-line-items__before"
     />
 
-    <mt-card
+    <sw-order-line-items-grid-sales-channel
         class="sw-order-create-general__line-items"
         position-identifier="sw-order-create-line-items"
```

Comment:
this position-identifier will not work anymore - maybe the `sw-order-line-items-grid-sales-channel` can accept it as a param and the placed in its mt-cart?